internal/scan-manager/core/src/main/resources/initial.sql

```diff
@@ -24,6 +24,14 @@ CREATE TABLE IF NOT EXISTS SCANNER(
             PRIMARY KEY (SCANNER_ID)
 )ENGINE INNODB;
 
+CREATE TABLE IF NOT EXISTS USER(
+            ID INT(10) NOT NULL AUTO_INCREMENT,
+            USERNAME VARCHAR(255),
+            EMAIL VARCHAR(256),
```

Comment:
It would be better if same lengths are used for similar data fields. Either 255 or 256.